### PR TITLE
Loading path data from JSON files.

### PR DESCRIPTION
**Pull request by F4talErr0r**
Core functionality implemented and structure defined. There is a room for additional code refactor but I would leave it as it is right now and address that during potential structure changes.

### CODE REVIEW

File: Assets/_Project/Resources/ExampleData/waypoints.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    ],
+    "navigation_audio": "audio.navigation.neptun"
+  },
+  {
+    "waypoint_id": "gliwice.rynek.ratusz",
+    "coordinates": [
+      50.29387,
+      18.66577
+    ],
+    "zone_name": "Rynek",
```
Comment (matadam667) [inline Assets/_Project/Resources/ExampleData/waypoints.json:25]: Do we even Zones? The last I heard they are completely gone.

File: Assets/_Project/Scripts/PathData/ExerciseSubcategory.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿namespace PolSl.UrbanHealthPath.PathData
+{
+    public enum ExerciseSubcategory
```
Comment (matadam667) [inline Assets/_Project/Scripts/PathData/ExerciseSubcategory.cs:3]: I don't know whether these subcatorogries are corrent. We need to get a list of exercises to figure it out I guess (?)

**Review by matadam667 — APPROVED** [review]
Besides a few questions about elements that might be outdated I can't find anything to complain about.

**Review by PaulinaCzapla — APPROVED** [review]

File: Assets/_Project/Resources/ExampleData/waypoints.json (hunk 1 above)
Comment (F4talErr0r) [reply Assets/_Project/Resources/ExampleData/waypoints.json:25]: I would leave them, I think they still might serve some purpose.

File: Assets/_Project/Scripts/PathData/ExerciseSubcategory.cs (hunk 2 above)
Comment (F4talErr0r) [reply Assets/_Project/Scripts/PathData/ExerciseSubcategory.cs:3]: Good point. However, updating them isn't big of a deal, so I'll merge current changes anyway.

**Pull request merged**